### ISSUE TITLE
handling.reset() and handling.isModified() isnt working right anymore

### ISSUE THREAD

**Issue by its4l3x** [altmp/altv-issues#1230]
### Description of the problem

When you want to reset the Handling via the .reset() function the reset function does nothing.

The same thing is happening with the isModified() function. The Function always returns false

### Reproduction steps

Execute the Following Snippet

Here a Snippet (Clientside):

```
let veh = alt.Player.local.vehicle;

veh.handling.tractionCurveMax = 10;
veh.handling.tractionCurveMin = 9.5;

console.log(veh.handling.isModified()) // -> console returns false

veh.handling.reset();
// -> handling is stil modified after that call
```

### Expected behaviour

the handling should be cleared and the isModified() property should return true

### Operating system

Windows

### Version

release/7.1

**Comment by Doxoh** [comment]
Can confirm:
https://www.file-upload.net/download-14765250/handling.zip.html

How to:

1. Spawn on Server
2. Enter into spawned vehicle
3. Open Chat (Key T)
4. Command /check -> check F8 log
5. Command /modify and then /check -> check F8 log
6. Command /reset and then /check ->  check F8 log

**Comment by Hutstaender** [comment]
Can confirm:
we just outcommented every handling system cause of this problem. It resulted in some... unexpected behaviour cause of multiple handling applyments with no reset like these clips. ;)

https://clips.twitch.tv/TsunderePlacidPineappleAllenHuhu-mVca0M3-Hk3BOO5S
https://clips.twitch.tv/HyperWittyChickenMrDestructoid-I90H7RWu4ubCYEIt
https://youtu.be/4m27ILgLWrY

**Comment by DurtyFree** [comment]
Should be fixed in latest dev (`dev/8.0-dev3`)  and might be merged into release after confirmation

**Comment by C0kkie** [comment]
Fixed in dev/8.0-dev3 & release 7.4
